### ISSUE TITLE
Test Can't launch in Chrome 75

### ISSUE THREAD

**Issue by dty0102** [cypress-io/cypress#5046]
### Current behavior:
When I try to run the test in Chrome, cypress opens the browser but can't launch the website correctly, it stays at this page until it gets timeout. 

<img width="1276" alt="Screen Shot 2019-08-27 at 3 07 40 PM" src="https://user-images.githubusercontent.com/11589347/63800657-99c9a480-c8dc-11e9-9114-9b975d8d3cc2.png">

Got this error in Chrome dev console

WebSocket connection to 'ws://localhost:57456/__socket.io/?EIO=3&transport=websocket' failed: Error during WebSocket handshake: net::ERR_INVALID_HTTP_RESPONSE
	../socket/node_modules/engine.io-client/lib/transports/websocket.js.WS.doOpen	@	cypress_runner.js:106667

### Desired behavior:
Should launch the website correctly as Electron

<img width="1272" alt="Screen Shot 2019-08-27 at 3 10 07 PM" src="https://user-images.githubusercontent.com/11589347/63800752-d1385100-c8dc-11e9-96d2-42618e935e7b.png">
 

### Versions
Chrome 75 and 76
Cypress 3.4.1
MacOS 10.12.6

**Comment by acomito** [comment]
I'm getting this too. Is there some additional setup required?

**Comment by sainthkh** [comment]
The time has passed and I hope the problem has been solved with Chrome 87+ and Cypress 6.2.0. If you have the same problem, please open a new issue with a reproducible example.
